packages/editor/src/dataviews/store/private-actions.ts

```diff
@@ -157,25 +157,24 @@ export const registerPostTypeSchema =
 			deletePost,
 			trashPost,
 			permanentlyDeletePost,
-		];
+		].filter( Boolean );
 
 		const fields = [
-			featuredImageField,
+			postTypeConfig.supports?.thumbnail &&
+				currentTheme?.[ 'theme-supports' ]?.[ 'post-thumbnails' ] &&
+				featuredImageField,
 			titleField,
 			authorField,
 			statusField,
 			dateField,
 			slugField,
```

Comment:
This one has a support key too.

Reply:
yes, you can check the different `<PostTypeSupportCheck` calls we have to find these.

Reply:
Hm.. the [docs for supports ](https://developer.wordpress.org/reference/functions/post_type_supports/)don't say that.. Are you sure there is such support?

I saw we have a `<PostTypeSupportCheck supportKeys="slug">` check, but I don't think it ever worked properly - unless there was such support in the past. Also I think the above code might not being used anywhere by quickly adding a breakpoint there. I'll check more to see if we can remove that code.

Reply:
I think you're right :)